### DISCUSSION TITLE
No sound output when iPhone is remotely connected to Mac

### DISCUSSION THREAD

**Discussion by Simondler** [rustdesk/rustdesk#10339] [General]
### Bug Description

No sound output when iPhone is remotely connected to MacBoth the iPhone and Mac have been restarted, but there is still no sound transmission

### How to Reproduce

No sound output when iPhone is remotely connected to Mac

### Expected Behavior

Fix this issue. Thanks

### Operating system(s) on local (controlling) side and remote (controlled) side

iphone13pm, iOS15.7

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

macos 15.2 (24C101)

### Screenshots

<img width="589" alt="截屏2024-12-22 11 39 22" src="https://github.com/user-attachments/assets/a22e4cbc-904e-4732-927c-0e246108225b" />

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/10230 fixed in 1.3.6 (not release yet)

**Comment by 21pages** [comment]
For iOS to hear Mac sound:

* Install the latest iOS RustDesk, with a build date on or after 2024-12-09.
* Ensure the Mac is ARM architecture and rustdesk 1.3.5 and set the audio input to "System sound."

Could you check if the conditions are met?

**Reply by Simondler** [reply]
Yes, that's what I did.
<img width="574" alt="截屏2024-12-22 12 20 34" src="https://github.com/user-attachments/assets/2a1a8562-622c-4921-8a72-739a63a8a437" />
<img width="363" alt="截屏2024-12-22 12 20 45" src="https://github.com/user-attachments/assets/7b30d245-61e8-4427-b3ff-16b1efd4638f" />
![IMG_2526](https://github.com/user-attachments/assets/e8024063-5782-43b4-afb6-13f9eef0de5d)

**Reply by rustdesk** [reply]
It works on our iphone.
